e2e/e2e-values.yml

```
@@ -0,0 +1,22 @@
+newrelic-infrastructure:
+  verboseLog: true
+  image:
+    pullPolicy: Never
+    tag: test
+    repository: test_image
```

Perhaps we can be a bit more specific here, and use something like
```suggestion
    tag: e2e
    repository: e2e/newrelic-infrastructure
```

I think this way it's more obvious which image we are testing